versioned_docs/version-2.0.x/reference/targets/archive.mdx

```diff
@@ -18,61 +18,62 @@ Backend: <span className="color--primary">``</span>
 ## `tags`
 
 <Field
-    type_repr={`field.type_hint`}
+    type_repr={`Iterable[str] | None`}
+    default_repr={`None`}
 >
 
 Arbitrary strings that you can use to describe a target. For example, you may tag some test targets with &#x27;integration_test&#x27; so that you could run `./pants --tags='integration_test' test ::` to only run on targets with that tag.
 
 </Field>
-
 ## `description`
 
 <Field
-    type_repr={`field.type_hint`}
+    type_repr={`str | None`}
+    default_repr={`None`}
```

Comment:
A lot of targets now pass this ``default_repr={`None`}``. Is that a problem?

Reply:
That was a bug in the old templates, it seems. See https://www.pantsbuild.org/docs/reference-archive